src/header/map.rs

```diff
@@ -29,8 +29,8 @@ pub use self::into_header_name::IntoHeaderName;
 /// # use http::header::{CONTENT_LENGTH, HOST, LOCATION};
```

Comment:
```suggestion
/// # use http::HeaderValue;
/// # use http::header::{CONTENT_LENGTH, HOST, LOCATION};
```

`HeaderValue` import is required to use `HeaderValue::from_static`. Rest looks good!